ietf/doc/views_search.py

```diff
@@ -425,106 +458,211 @@ def ad_dashboard_sort_key(doc):
 
     return "3%s" % seed
 
+
 def ad_workload(request):
+    delta = datetime.timedelta(days=30)
+    today = datetime.date.today()
```

Comment:
For better timezone behavior (and compatibility with the timezone-aware work going on right now), it'd be better to use `right_now = timezone.now()` (imported with `from django.utils import timezone`) instead of computing `today()`.

(other changes will be needed, noted inline)

```suggestion
    right_now = timezone.now()
```